docker/prepare-dev-environment/solr/docker-compose.yml

```diff
@@ -18,8 +18,6 @@ services:
     environment:
       JAVA_TOOL_OPTIONS: "-Dfile.encoding=UTF8"
       GRADLE_RO_DEP_CACHE: /gradle-ro-dep-cache
-      ZK_HOSTS: "${PROJECT_NAME}-${SOLR_CLOUD_ZK_CONTAINER_1_NAME}:2181,${PROJECT_NAME}-${SOLR_CLOUD_ZK_CONTAINER_2_NAME}:2181,${PROJECT_NAME}-${SOLR_CLOUD_ZK_CONTAINER_3_NAME}:2181"
-      SOLR_HOSTS: "http://${PROJECT_NAME}-${SOLR_CLOUD_CONTAINER_1_NAME}:8983/solr,http://${PROJECT_NAME}-${SOLR_CLOUD_CONTAINER_2_NAME}:8983/solr"
       SOLR_HOST: ${PROJECT_NAME}-${SOLR_CLOUD_CONTAINER_1_NAME}:8983
```

Comment:
You can probably remove this environment variable too, as it is not used with Solr8

Reply:
ok, I will test and remove

Reply:
If I remove `SOLR_HOST: ${PROJECT_NAME}-${SOLR_CLOUD_CONTAINER_1_NAME}:8983`

Again I'm getting the below error:
```

solr-populator-1           | curl: (6) Could not resolve host: solr
solr-populator-1           | curl: (6) Could not resolve host: solr
solr-populator-1           | curl: (6) Could not resolve host: solr
```

Reply:
I think it is because you removed these variables: `ZK_HOSTS` and `SOLR_HOSTS` and they are used in these lines:
```
./gradlew -PdataFilesLocation=/root \
        -PexperimentFilesLocation=/atlas-data/exp \
        -PexperimentDesignLocation=/atlas-data/expdesign \
        -PzkHosts=$${ZK_HOSTS} \
        -PsolrHosts="" \
        -PjdbcUrl=jdbc:postgresql://${POSTGRES_HOST}:5432/${POSTGRES_DB} \
        -PjdbcUsername=${POSTGRES_USER} \
        -PjdbcPassword=${POSTGRES_PASSWORD} \
        -PzkHosts=$${ZK_HOSTS} \
        -PsolrHosts=$${SOLR_HOSTS} \
        :cli:bootRun --args="bulk-analytics-json --output=/root/experiments-jsonl -e $(echo ${EXP_IDS} ${PRIVATE_EXP_IDS} | sed -e "s/ /,/g")"
```